examples/flutter_gallery/lib/demo/cupertino/cupertino_buttons_demo.dart

```diff
@@ -49,8 +49,10 @@ class _CupertinoButtonDemoState extends State<CupertinoButtonsDemo> {
                       CupertinoButton(
                         child: const Text('Cupertino Button'),
                         onPressed: () {
-                          setState(() { _pressedCount += 1; });
-                        }
+                          setState(() {
```

Comment:
what do people think of a general quick fix to make these sorts of methods use arrow syntax. It seems cleaner and dartfmt handles it the way you would want today.
For example:
onPressed: () {
  setState(() => _pressedCount += 1);
}
I believe the reason for not doing that is historic as previously the return type was problematic so a quick fix is reasonable.

Reply:
Perhaps the same explanation as https://github.com/flutter/flutter/pull/8595#discussion_r104559527

> [...]. We're trying to make setState look like a block statement, where it's "obvious" that the statements inside the block are executed immediately. This change hides what's going on, it makes it look like we're taking a reference to call later or something.

Reply:
Ok sounds like the cleanup for this case would need to be done manually or at minimum avoid setState.

Reply:
yeah for setState specifically I really don't recommend using `=>`, it confuses the issue a lot.

But in general I would reserve `=>` for an actual non-void return. `=>` means `return` and it seems confusing to also use it to mean "short statement".

Reply:
I think this has come up already, but. I think the the `=>` syntax should be reserved for one-liners that return a value.